modules/preferences/test/spec.coffee

```diff
@@ -5,6 +5,14 @@ describe 'hx-preferences', ->
       expect(hx.preferences.supportedLocales(list)).toEqual(hx.preferences)
       expect(hx.preferences.supportedLocales()).toEqual(list)
 
+    it 'locale: should not be possible to explicitly clear the locale', ->
+      # sanity check
+      expect(hx.preferences.locale()).toBeDefined()
+
+      hx.preferences.locale undefined
+      expect(hx.preferences.locale()).toBeDefined()
```

Comment:
Discussed this with @jmsmyth and he agrees that rather than doing nothing, this should use the `navigator.language`
meaning that the following is true:
```
hx.preferences.locale(undefined)
hx.preferences.locale().should.equal(navigator.language)
```